.github/workflows/docker.yml

```diff
@@ -333,7 +333,7 @@ jobs:
       - name: Create and Push Manifest
         id: manifest
         if: github.repository == 'netdata/netdata'
-        run: docker buildx imagetools create $(.github/scripts/gen-docker-imagetool-args.py /tmp/digests '' ${{ needs.gen-tags.outputs.tags }})
+        run: docker buildx imagetools create "$(.github/scripts/gen-docker-imagetool-args.py /tmp/digests '' "${{ needs.gen-tags.outputs.tags }}")"
```

Comment:
This change broke the workflow. Publishing the latest/edge images has failed for the last 4 days.

`docker buildx imagetools create` exits with "ERROR: no sources specified".

Because the `.github/scripts/gen-docker-imagetool-args.py` script returns a list of parameters, we shouldn't `"` its output.

Reply:
Fixes in #17312